migrate-record-set.sh

```diff
@@ -10,7 +10,9 @@ cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 || exit
 }
 
 export AWS_PROFILE=di-authentication-"${2}"-AWSAdministratorAccess
-aws sso login --profile "${AWS_PROFILE}"
+if ! aws sts get-caller-identity &> /dev/null; then
```

Comment:
Not sure this'll work. `aws sts get-caller-identity` returns 0 even if user not logged in

Reply:
```sh
❯ echo PROFILE="${AWS_PROFILE}"; if aws sts get-caller-identity &>/dev/null ; then echo "valid"; else echo "invalid"; fi
PROFILE=
invalid
❯ export AWS_PROFILE=di-auth-development-admin
❯ echo PROFILE="${AWS_PROFILE}"; if aws sts get-caller-identity &>/dev/null ; then echo "valid"; else echo "invalid"; fi
PROFILE=di-auth-development-admin
valid
❯ aws sso logout
❯ echo PROFILE="${AWS_PROFILE}"; if aws sts get-caller-identity &>/dev/null ; then echo "valid"; else echo "invalid"; fi
PROFILE=di-auth-development-admin
invalid
```